crates/uv/tests/pip_show.rs

```diff
@@ -500,7 +513,75 @@ fn show_editable() -> Result<()> {
     Name: poetry-editable
     Version: 0.1.0
     Location: [WORKSPACE_DIR]/site-packages
+    Editable project location: [EDITABLE_INSTALLS_PREFIX]poetry_editable
     Requires: anyio
+    Required-by:
```

Comment:
I wonder if we should omit these when they're empty

Reply:
I followed the approach used by pip. I have no preference, either option is fine.

Reply:
I'd vote to keep pip bwhaviour, otherwise users can wonder if it is even implemented.

Reply:
I don't think being worried that people will wonder if it it's implemented makes sense as a long-term choice. It seems likely if you care about this feature that you'll have at least one package with `Required-by` set. I don't have strong feelings but I'd lean towards omitting empty fields. Curious for others' thoughts though cc @konstin

Reply:
I don't have an opinion on that, we could add `(none)` as placeholder if it helps